code/modules/reagents/chemistry/reagents/alcohol.dm

```diff
@@ -1544,18 +1544,18 @@
 	var/light_activated = FALSE
 
 /datum/reagent/consumable/ethanol/sontse/on_mob_life(mob/living/M)
-	if(ismoth(M))
-		if(current_cycle == 5)
-			to_chat(M, "<span class='warning'>The Sun was within you all this time!</span>")
-			if(!light_activated)
-				M.set_light(2)
-				light_activated = TRUE
+	if(current_cycle != 5 || !ismoth(M))
+	to_chat(M, "<span class='warning'>The Sun was within you all this time!</span>")
+	if(!light_activated)
+		M.set_light(2)
```

Comment:
missed a return.

Reply:
somehow you reviewed stuff between commits =)

Reply:
oops